api/internal/handler/ssl/ssl.go

```diff
@@ -198,7 +198,16 @@ func (h *Handler) Create(c droplet.Context) (interface{}, error) {
 		return handler.SpecCodeResponse(err), err
 	}
 
-	return ret, nil
+	//format respond
+	_ssl := &entity.SSL{}
+	err = utils.ObjectClone(ret, _ssl)
```

Comment:
@Jaycean @nic-chen @starsz  How about use type assertion here like the following?

```go
ssl := ret.(*entity.SSL)
```

I also noticed that current `Get` method also uses `ObjectClone()`. I'm a little confused and looking forward to your directions. Thank you very much.

Reply:
Oh,  I only know the `Get` method.
Because the ret that return by the Get is a point. We will hide the key in the latter (see line 207 ~ 208). If we don't do the objectClone, then will influence the value in s.cache.

Reply:
In fact, we can only get the value instead of using `ObjectClone()`

```
ssl := ret.(*entity.SSL)
_ssl = *ssl
```

Reply:
> Oh, I only know the `Get` method.
> Because the ret that return by the Get is a point. We will hide the key in the latter (see line 207 ~ 208). If we don't do the objectClone, then will influence the value in s.cache.

@starsz thanks for your explanations. Agreed with you for getting the value instead of cloning. Maybe more comments could be added here for noticing other developers. As far as I know, we could directly manipulate the returned value of Create/Update/Patch methods without any side effect. Please correct me if I was wrong.

Reply:
> we could directly manipulate the returned value of Create/Update/Patch methods without any side effect.

Yes.

Reply:
> > we could directly manipulate the returned value of Create/Update/Patch methods without any side effect.
> 
> Yes.

Thanks 👍

Reply:
@Jaycean  please directly manipulate the returned value instead of cloning within Create/Update/Patch methods.

Reply:
OK, thks.